setup.py

```diff
@@ -59,6 +58,9 @@
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6'
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```

Comment:
Nit: this probably works, but it's not one of the targets in our travis tests

Reply:
would be very surprising if it didn't work. do you think it's worth adding 3.8 to travis?

Reply:
Yeah sure. These tests are very fast and run infrequently so it's unlikely to hurt our travis job credits or whatever

Reply:
https://github.com/dimagi/jsonobject/pull/180